store/sqlstore/channel_store.go

```diff
@@ -914,11 +914,11 @@ func (s SqlChannelStore) GetChannels(teamId string, userId string, includeDelete
 	_, err := s.GetReplica().Select(channels, query, map[string]interface{}{"TeamId": teamId, "UserId": userId})
 
 	if err != nil {
-		return nil, model.NewAppError("SqlChannelStore.GetChannels", "store.sql_channel.get_channels.get.app_error", nil, "teamId="+teamId+", userId="+userId+", err="+err.Error(), http.StatusInternalServerError)
+		return nil, errors.Wrapf(err, "failed to get channels with TeamId=%s and UserId=%s", teamId, userId)
 	}
 
 	if len(*channels) == 0 {
-		return nil, model.NewAppError("SqlChannelStore.GetChannels", "store.sql_channel.get_channels.not_found.app_error", nil, "teamId="+teamId+", userId="+userId, http.StatusBadRequest)
+		return nil, store.NewErrInvalidInput("Channel", "UserId", userId)
```

Comment:
I think this should be a notfound error rather than invalid input. The error code returned is rather incorrect.

Reply:
Ok I will change this